src/include/OSL/platform.h

```diff
@@ -499,6 +500,24 @@
 
 OSL_NAMESPACE_ENTER
 
+
```

Comment:
If you compiler/headers do support c++20 and not compiling for CUDA, should this conditionally compiled to be
template <typename To, typename From>
using bitcast = std::bitcast<To, From>;

Reply:
Yeah, but that's the tricky part -- in addition to very few users ready to rely on C++20, the std one still wouldn't have the HOSTDEVICE decorators, so I'm not sure the extra clutter of conditionally choosing between this and the std one gives us any tangible advantage. To the best of my knowledge, this code optimizes to the same thing anyway.

Reply:
Ok, it could always be added later should there be a need